catalog/spatial/registry/registry-rest-endpoint/src/main/resources/templates/report.hbt

```diff
@@ -82,7 +82,7 @@
             margin-left: 50px
         }
     </style>
-    <link href="/services/registries/assets/css/bootstrap-theme.min.css" rel="stylesheet" />
+    <link href="/services/internal/registries/assets/css/bootstrap-theme.min.css" rel="stylesheet" />
 </head>
```

Comment:
this has to be modified in all the report handlebar templates `report, summary, organizations, and error` also there is a `report.css` class as a link too